jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -493,6 +493,16 @@ static void completeInitialization() {
 		throw new InternalError(e.toString());
 	}
 	/*[ENDIF]*/	//!Sidecar19-SE_RAWPLUSJ9&!Sidecar18-SE-OpenJ9
+
+	/*[IF JAVA_SPEC_VERSION >= 19]*/
+	try {
+		java.lang.reflect.Field notifyJvmtiEvents = VirtualThread.class.getDeclaredField("notifyJvmtiEvents"); //$NON-NLS-1$
```

Comment:
`java.lang.reflect.Field` is also used at two other places. It will be cleaner to `import java.lang.reflect.Field;` and just use `Field`.